# Increased PLAYERBUFSIZE to (MAXFRAMESIZE * 5) to avoid stuttering replay under heavy
# system load, and to better document that this buffer size is related to the maximum
# frame size.
#
--- a/dvbplayer.c	2017/11/26 14:55:03	5.0
+++ b/dvbplayer.c	2019/05/27 13:54:19
@@ -230,7 +230,7 @@
 
 // --- cDvbPlayer ------------------------------------------------------------
 
-#define PLAYERBUFSIZE  MEGABYTE(1)
+#define PLAYERBUFSIZE  (MAXFRAMESIZE * 5)
 
 #define RESUMEBACKUP 10 // number of seconds to back up when resuming an interrupted replay session
 #define MAXSTUCKATEOF 3 // max. number of seconds to wait in case the device doesn't play the last frame
